tests/fixtures/landsat-8-l1-collection.json

```diff
@@ -1,5 +1,6 @@
 {
     "id": "landsat-8-l1",
+    "type": "Collection",
```

Comment:
all of these fixtures were missing type, and therefore were invalid stac collections